pkg/daemon/daemon.go

```diff
@@ -333,6 +335,11 @@ func (d *Daemon) sharedErofsUmount(rafs *Rafs) error {
 		return errors.Wrapf(err, "umount erofs %s mountpoint, %s", err, mountpoint)
 	}
 
+	// delete fscache bootstrap cache file
+	if err := c.UnbindBlob("", fscaheID); err != nil {
```

Comment:
Does this PR depend on changes in nydusd?

Reply:
Yes , this depend on changes for nydusd API , so maybe we should review the PR on nydusd first? and make API fixed.
https://github.com/dragonflyoss/image-service/pull/894

Reply:
Sure. Let's wait for that dependency and delay this PR for snapshotter v0.5